SimCalorimetry/EcalEBTrigPrimProducers/python/ecalEBTriggerPrimitivePhase2Digis_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+# Trigger Primitive Producer
+from SimCalorimetry.EcalEBTrigPrimProducers.ecalEBTriggerPrimitivePhase2Digis_cfi import *
+
+
+# esmodule creating  records + corresponding empty essource
+# when commented, one takes the configuration from the global tag
+#
+
+
+#Common
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
```

Comment:
Why use the hgcal modifier if there is a `phase2_ecalTP_devel` one?

Reply:
Right. That was from the very first attempts to run this stuff, when ecal_devel was not existing yet, I think. Anyway just replaced with from 
Configuration.Eras.Modifier_phase2_ecalTP_devel_cff import phase2_ecalTP_devel
phase2_ecalTP_devel.toModify( simEcalEBTriggerPrimitivePhase2Digis)